### ISSUE TITLE
[COS] Crash on Dedicated Server

### ISSUE THREAD

**Issue by drakray** [r8420/MoreOverlays-updated#6]
```
[29Oct2020 15:17:52.525] [main/FATAL] [net.minecraftforge.fml.loading.RuntimeDistCleaner/DISTXFORM]: Attempted to load class net/minecraft/client/gui/screen/Screen for invalid dist DEDICATED_SERVER
[29Oct2020 15:17:52.526] [main/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Failed to load class at.feldim2425.moreoverlays.MoreOverlays
java.lang.RuntimeException: Attempted to load class net/minecraft/client/gui/screen/Screen for invalid dist DEDICATED_SERVER
	at net.minecraftforge.fml.loading.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:71) ~[forge-1.16.3-34.1.25.jar:34.1]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:85) ~[modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:119) ~[modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader$DelegatedClassLoader.findClass(TransformingClassLoader.java:256) ~[modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:135) ~[modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:97) ~[modlauncher-7.0.1.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_251]
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_251]
	at java.lang.Class.forName(Unknown Source) ~[?:1.8.0_251]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.<init>(FMLModContainer.java:61) ~[?:34.1]
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_251]
	at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.8.0_251]
	at net.minecraftforge.fml.javafmlmod.FMLJavaModLanguageProvider$FMLModTarget.loadMod(FMLJavaModLanguageProvider.java:73) ~[?:34.1]
	at net.minecraftforge.fml.ModLoader.buildModContainerFromTOML(ModLoader.java:292) ~[?:?]
	at net.minecraftforge.fml.ModLoader.lambda$buildMods$29(ModLoader.java:272) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source) ~[?:1.8.0_251]
	at java.util.HashMap$EntrySpliterator.forEachRemaining(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.ReferencePipeline.collect(Unknown Source) ~[?:1.8.0_251]
	at net.minecraftforge.fml.ModLoader.buildMods(ModLoader.java:274) ~[?:?]
	at net.minecraftforge.fml.ModLoader.lambda$gatherAndInitializeMods$11(ModLoader.java:176) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source) ~[?:1.8.0_251]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:1.8.0_251]
	at java.util.stream.ReferencePipeline.collect(Unknown Source) ~[?:1.8.0_251]
	at net.minecraftforge.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:178) ~[?:?]
	at net.minecraftforge.fml.server.ServerModLoader.load(ServerModLoader.java:44) ~[?:?]
	at net.minecraft.server.Main.main(Main.java:96) ~[?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_251]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_251]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_251]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_251]
	at net.minecraftforge.fml.loading.FMLServerLaunchProvider.lambda$launchService$0(FMLServerLaunchProvider.java:51) ~[forge-1.16.3-34.1.25.jar:34.1]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-7.0.1.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-7.0.1.jar:?]
	at net.minecraftforge.server.ServerMain$Runner.runLauncher(ServerMain.java:63) [forge-1.16.3-34.1.25.jar:?]
	at net.minecraftforge.server.ServerMain$Runner.access$100(ServerMain.java:60) [forge-1.16.3-34.1.25.jar:?]
	at net.minecraftforge.server.ServerMain.main(ServerMain.java:57) [forge-1.16.3-34.1.25.jar:?]
```

https://pastebin.com/6rQzxJe3

**Comment by drakray** [comment]
MC 1.16.3
Forge 34.1.25
MoreOverlays 1.18.9_PREVIEW_1-mc1.16.3

**Comment by r8420** [comment]
The mod is client-side only and doesn't have to be on the server.

**Comment by drakray** [comment]
If it is not installed on the server, but on the client, the server list show the big red X, saying "Incompatible FML modded server[lf]Server mod list not compatible"
The client can connect to it though

**Comment by r8420** [comment]
Should be fixed in version 1.18.10: [https://www.curseforge.com/minecraft/mc-mods/more-overlays-updated/files/3099374](https://www.curseforge.com/minecraft/mc-mods/more-overlays-updated/files/3099374)
